mmv1/products/cloudfunctions/CloudFunction.yaml

```diff
@@ -0,0 +1,186 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
This and probably other files are missing the first line: 
```
# Copyright {YEAR} Google Inc.
```
which I believe is leading to a lint error

Reply:
it was missing in connection, and it fixed.